docs/rest/DaVinciFormulary/DaVinciFormulary_Sample_Resources.http

```diff
@@ -0,0 +1,2183 @@
+# This file includes sample resources that you can use to test DaVinci Formulary
+### REST Client
+@fhirurl= <FHIR URL>
+@clientid= <CLIENT ID>
+@clientsecret= <CLIENT SECRET>
+@tenantid= <SUBSCRIPTION ID>
+@contentType = application/json
+
+### Get Metadata - Allows you to get the current capability statement
+# this will update as you add search parameters and profiles to the database
+GET {{fhirurl}}/metadata
+
+### Get access token
+# @name getAADToken
+POST https://login.microsoftonline.com/{{tenantid}}/oauth2/token
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials
+&resource={{fhirurl}}
+&client_id={{clientid}}
+&client_secret={{clientsecret}}
+
+### 
+GET {{fhirurl}}/StructureDefinition?url=http://hl7.org/fhir/us/davinci-drug-formulary/StructureDefinition/usdf-FormularyDrug
+Authorization: Bearer {{token}}
+content-type: {{contentType}}
+
+### Capture access token from getToken request
+@token={{getAADToken.response.body.access_token}}
+
+
+### Variables for creating resources
+# Add values to each ID as you create them.
+# You need to create in order and add IDs as you go or references will not work
+
+@formularydrugV1002=<formularydrugV1002>
+@FormularyDrugV3001=<FormularyDrugV3001>
+@cmsip9=<cmsip9>
+@CoveragePlanV1002=<CoveragePlanV1002>
+@CoveragePlanV3001=<CoveragePlanV3001>
+@CoveragePlanV3002=<CoveragePlanV3002>
+@CoveragePlanV3004t=<CoveragePlanV3004t>
+@FormularyDrugV3002=<FormularyDrugV3002>
+
+### Create formularydrugV1002
+POST {{fhirurl}}/MedicationKnowledge
+Authorization: Bearer {{token}}
+content-type: {{contentType}}
+
```

Comment:
Can you move the POST bodies to separate files so that they are easier to read?

Reply:
@LTA-Thinking - If Moira uses your method above, does that mean it will know about the variables? I had the issue where if the bodies were in separate files the variables for linking didn't get set.

Reply:
@CaitlinV39 I don't know. I haven't experimented with using variables in POST bodies. If that can't be done then keeping them in the same file makes sense.

Reply:
Didn't address this issue, can look into further once we have passed all the CMS tests

Reply:
Found this in the docs for future reference: 
If you want to use variables in that file, you'll have to use an @ to ensure variables are processed when referencing a file (UTF-8 is assumed as the default encoding)

POST https://example.com/comments HTTP/1.1
Content-Type: application/xml
Authorization: token xxx

<@ ./demo.xml